### DISCUSSION TITLE
Timeline: 给设置的 dot 设置一个自定义图标时，图标后面会有一个白色的背景

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#41056] [General]
-![image](https://user-images.githubusercontent.com/6016942/222934716-2e1597a3-7751-4382-b862-4a0d5a651fed.png)

<Timeline
items={[
   { dot: <SettingsOutlined color='red' />, children: 'test' }
]}
/-

https://codesandbox.io/s/antd-reproduction-template-forked-jyh2k9?file=/index.js

看元素结构，是 timeline 生成的元素的背景，不是图标的背景， 我本地本身使用的是 antd-design-icons 里的图标

**Comment by Yuiai01** [comment]
使用的图标是内置的还是从外部引入的呢？感觉像是没有选择透明背景，可以提供一个可以复现问题的在线链接么～

**Comment by kiner-tang** [comment]
你好 @ekuku1984, 我们需要你提供一个在线的重现实例以便于我们帮你排查问题。你可以通过点击这里创建一个 [antd@5.x](https://u.ant.design/codesandbox-repro) 或 [antd@4.x](https://u.ant.design/codesandbox-repro-4x) 的 codesandbox，或者提供一个最小化的 GitHub 仓库。

![](https://gw.alipayobjects.com/zos/antfincdn/y9kwg7DVCd/reproduce.gif)
